packages/meteor/message-dispatch.js

```diff
@@ -1,59 +0,0 @@
-// This code receives, dispatches, and responds to inter-process messages
-// sent by the parent (build) process. See tools/runners/run-app.js for
-// the other side of this communications system.
-
-// The process.send method is only defined when the current process was
-// spawned by another process with an IPC channel.
-if (typeof process.send === "function") {
-  const promises = Object.create(null);
-
-  // Listen for messages from the parent process and dispatch them to the
-  // appropriate package, as identified by packageName. To receive these
-  // messages, packages should export an onMessage function that takes the
-  // payload as a parameter.
-  process.on("message", ({
-    type = "FROM_PARENT",
-    responseId = null,
-    packageName,
-    payload,
-  }) => {
-    if (type === "FROM_PARENT" &&
-        typeof packageName === "string") {
-      // Keep the messages and their responses strictly ordered per
-      // package, one after the last. The first message waits for the
-      // package to call Package._define(packageName, exports).
-      promises[packageName] = (
-        promises[packageName] || Package._promise(packageName)
```

Comment:
Since we're no longer using `Package._promise` in core, and it was only recently introduced for this (now removed) code, I think we should remove it for the sake of keeping the APIs exposed by the core `meteor` package as small as possible.